pom.xml

```diff
@@ -54,6 +58,31 @@
             <artifactId>log4j-core</artifactId>
             <version>2.11.1</version>
         </dependency>
+     
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.4.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.4.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
I don't think you will need both versions of junit. Use just jupiter, and delete this one.